lib/one_model.ex

```diff
@@ -778,12 +805,12 @@ defmodule OneModel do
   end
 
   def paging_stats(params, fun) do
-    offset = params |> query_params() |> Map.get(:offset, 0)
+    offset = %{} |> offset_params(params) |> Map.get(:offset, 0)
     Keyword.put(fun.(), :offset, offset)
   end
 
   def get_query_count(model, params, opts \\ []) do
-    query_params = query_params(params)
+    query_params = query_params(%{}, params)
```

Comment:
I don't understand this change. You added `query_params` by I don't see a change for it's use. Can can only presume this was broken before your fix, but then it should have been emitting a compiler warning/error about query_params/0 function  call with out parens??

Reply:
There are two functions that are essentially independent, but with the same name. query_params/1 examines all the parameters of the overall query and returns a map of the four things that it looks at (count, offset, sort and query). query_params/2 specifically looks at the `query` option in the parameters. 

This function, and the previous one, `paging stats`, both called the arity 1 version but really only needed to call one of the four functions that it calls. By changing both functions to call the function specific to their area of interest I was able to avoid the three logs being generated in the test `sort json invalid` and cut it back to only generating one log. 

So, the nature of this change is that instead of calling the /1, which calls the /2 and three other unneeded functions, I just call the /2 directly and avoid the overhead, and the extra logging because the sort parameter is invalid in that specific test.